workbench-core/eslint-custom/src/custom-eslint.ts

```diff
@@ -7,7 +7,18 @@
 import { importConvention } from './rules/import-convention';
 import { namingConvention } from './rules/naming-convention';
 
-const rules: any = Object.assign({}, namingConvention.rules, importConvention.rules);
+const rules: any = Object.assign({}, namingConvention.rules, importConvention.rules, {
+  '@typescript-eslint/no-unused-vars': ['error', { argsIgnorePattern: '^_' }],
+  '@typescript-eslint/naming-convention': [
```

Comment:
This can go in here : https://github.com/aws-solutions/solution-spark-on-aws/blob/topic/allowUnusedParamsHavingLeadingUnderscore/workbench-core/eslint-custom/src/rules/naming-convention.ts